mw-webapp/src/component/input/Input.tsx

```diff
@@ -0,0 +1,39 @@
+import {FC, HTMLInputTypeAttribute} from "react";
+import cn from "classnames";
+import styles from "./Input.module.scss";
+
+
+interface InputProps {
+  type?: HTMLInputTypeAttribute;
+  placeholder?: string;
+  className?: string | Array<string>;
+  inputMode?: "none" | "text" | "tel" | "url" | "email" | "numeric" | "decimal" | "search" | undefined;
+  disabled?: boolean;
+  required?: boolean;
+}
+
+export const Input: FC<InputProps> = ({
+  type = "text",
+  placeholder = "",
+  className = "",
+  disabled = false,
+  required = false,
+  inputMode,
+}) => {
+  return (
+    <div className={styles.input__wrapper}>
+      <label
+        className={styles["visually-hidden"]}
+      >
+        {placeholder}
+      </label>
+      <input type={type}
+        placeholder={placeholder}
+        className={cn(styles.input, [className])}
+        inputMode={inputMode}
+        disabled={disabled}
+        required={required}
```

Comment:
I expect to see `value` and `onChange` props. Could you provide an example of how to work with the current input, or it could be used only with forms?

Reply:
My mistake. I completely forgot about events. I'm sorry